app/templates/partials/answers/Minute.html

```diff
@@ -0,0 +1,3 @@
+{% with input_type = 'mins' %}
+{% include 'partials/forms/input-type.html' %}
```

Comment:
'input-type' seems like quite a vague name to give this partial given we already have one named 'input'. Isn't it really an integer-field?